webapp/settings.py

```diff
@@ -2,6 +2,9 @@
 Django project settings
 """
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
```

Comment:
I just moved the line. I have never really questioned it as it seems to be a standard addition do Django settings files and seemed appropriate.